flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/strategy/TestingSchedulingExecutionVertex.java

```diff
@@ -42,6 +44,7 @@ public class TestingSchedulingExecutionVertex implements SchedulingExecutionVert
 
     private final Map<IntermediateResultPartitionID, TestingSchedulingResultPartition>
             resultPartitionsById;
+    private final List<IntermediateDataSetID> cachedIntermediateDataSetID;
```

Comment:
cachedIntermediateDataSetID -> consumedCachedIntermediateDataSetId

As once explained, we need to clarify it is a produced partition or a consumed partition to avoid confusion.
Similar changes should also be applied around.